data/mods/Xedra_Evolved/monsters/changeling.json

```diff
@@ -0,0 +1,365 @@
+  },
+  {
+    "id": "mon_fetch_child",
+    "type": "MONSTER",
+    "name": { "str": "fetch, childlike", "str_pl": "childlike fetches" },
+    "description": "It looks like either a child or a child sized doll the sharp objects in its hands and the reflective nature of its eyes are the real tells this isn't just a living child or doll.  You'd be hard-pressed to not feel like you were killing an actual child by putting it down.",
+    "default_faction": "changeling",
+    "bodytype": "human",
+    "species": [ "CHANGELING" ],
+    "volume": "30000 ml",
+    "weight": "40750 g",
+    "hp": 40,
+    "speed": 65,
+    "material": [ "fae_flesh" ],
+    "symbol": "f",
+    "color": "light_green",
+    "aggression": 60,
+    "morale": 100,
+    "melee_skill": 2,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 2,
+    "armor_elec": 1,
+    "armor_bullet": 4,
+    "dodge": 2,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_child_body", "wps_humanoid_head_big" ],
+    "vision_day": 30,
+    "vision_night": 3,
+    "harvest": "demihuman",
+    "special_attacks": [ { "type": "melee", "id": "sharp_objects", "attack_upper": false, "cooldown": 15 }, [ "PARROT_AT_DANGER", 800 ] ],
+    "death_drops": { "subtype": "collection", "groups": [ [ "default_zombie_children_clothes", 100 ], [ "child_items", 65 ] ] },
+    "death_function": { "effect": { "id": "death_guilt", "min_level": 9 } },
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES", "GUILT" ]
+  },
+  {
+    "id": "mon_fetch_daffodil_warrior",
+    "type": "MONSTER",
+    "name": { "str": "daffodil warrior" },
+    "description": "Wispy and willowy these fetches are the cannon fodder of the Fae.  Said to grow in vast fields under the hill and harvested right before a battle legends says they live but a day.  It's features are quite plant-like and one arm ends in an ornate club.",
+    "default_faction": "changeling",
+    "bodytype": "human",
+    "species": [ "CHANGELING" ],
+    "volume": "45000 ml",
+    "weight": "60750 g",
+    "hp": 20,
+    "speed": 75,
+    "material": [ "fae_flesh" ],
+    "symbol": "d",
+    "color": "light_green_yellow",
+    "aggression": 60,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 9,
+    "armor_cut": 4,
+    "armor_bullet": 4,
+    "dodge": 2,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_body_armor", "wps_humanoid_open_helmet" ],
+    "vision_day": 30,
+    "vision_night": 3,
+    "death_function": { "message": "The %'s body 'poufs' and bursts into a rain of pollen!", "corpse_type": "NO_CORPSE" },
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES" ]
+  },
+  {
+    "id": "mon_fae_wolf",
+    "type": "MONSTER",
+    "name": { "str": "shimmer wolf" },
+    "description": "This is some form of otherworldly wolf.  It's luxurious fur shimmers in the light.  If an Irish Wolfhound had an ancestor that couldn't be bothered to mingle with mere mortal men it would be this wolf.",
```

Comment:
```suggestion
    "description": "This is some form of otherworldly wolf.  Its luxurious fur shimmers in the light.  If an Irish Wolfhound had an ancestor that couldn't be bothered to mingle with mere mortal men it would be this wolf.",
```